### PR TITLE
Complete new nav

### PR DESCRIPTION
**Pull request by Hidde-Heijnen**
completed @BenjaminMeijs Nav dropdown and mobile nav

### CODE REVIEW

File: src/components/layout/Navbar.tsx
Comment (ThijnK) [inline src/components/layout/Navbar.tsx:74]: Please use the `TOKENS` variable for the symbol of the SECOIN.
Also, we should decide on a consistent way to either always use $ in front of the symbol, or not (currently we do not, but can change if you so desire)

File: src/components/layout/Navbar.tsx
Comment (ThijnK) [inline src/components/layout/Navbar.tsx:80]: Same here as above comment for the SECOIN and SECOREP, use `TOKENS.secoin.symbol` and `TOKENS.rep.symbol`

**Review by ThijnK — CHANGES_REQUESTED** [review]
Few minor changes, see comments

File: src/components/layout/Navbar.tsx
Comment (ThijnK) [inline src/components/layout/Navbar.tsx:75]: There's a period at the end of this description, while there isn't one in the description of Mining page.
You know what they say, it's all in the details

File: src/components/ui/Sheet.tsx
Comment (ThijnK) [inline src/components/ui/Sheet.tsx:16]: I know you love Next.js, but alas, we're not using it, so no need for this

File: src/components/layout/Navbar.tsx
Comment (ThijnK) [inline src/components/layout/Navbar.tsx:91]: What happened to naming convention for components??
Same for the other components in this file

File: src/components/layout/Navbar.tsx
Comment (ThijnK) [inline src/components/layout/Navbar.tsx:266]: Why is this suddenly not an arrow function, while all other components are?

File: src/components/layout/Navbar.tsx
Comment (ThijnK) [inline src/components/layout/Navbar.tsx:164]: Accidental hard coded value?

**Review by ThijnK — CHANGES_REQUESTED** [review]
See comments above

**Comment by Hidde-Heijnen** [comment]
> See comments above

Will fix today, after merge to main + fix env on vercel?

File: src/components/layout/Navbar.tsx
Comment (Hidde-Heijnen) [reply src/components/layout/Navbar.tsx:91]: Dat was @BenjaminMeijs whoops, anders had de naam conflict met de types

**Comment by Hidde-Heijnen** [comment]
Check now @ThijnK

**Review by ThijnK — COMMENTED** [review]
LGTM!

**Pull request merged**
